Kernel/BottomUpEvaluation.hpp

```diff
@@ -368,17 +370,23 @@ class BottomUpEvaluation {
 
     ASS(recResults->size() == 1);
     auto result = recResults->pop();
-    DEBUG("eval result: ", toEval, " -> ", result);
+    DEBUG_BOTTOM_UP(0, "eval result: ", toEval, " -> ", result);
     return result;
   }
 };
 
 
 }
+#undef DEBUG
 
 #include "Kernel/Term.hpp"
 
 namespace Lib {
+
+struct TermListContext {
+  bool ignoreTypeArgs = true;
```

Comment:
I'm OK with this solution but I'm not sure I see where this is ever false - everything seems to use the default constructor.

Reply:
Ahah, but you want to give people the option, right.